### PR TITLE
Edu QLD Hand Precursive: Version 2.000 added



### PR DESCRIPTION
**Pull request by simoncozens**
Taken from the upstream repo https://github.com/SorkinType/QLDSchoolHandAustralia at commit https://github.com/SorkinType/QLDSchoolHandAustralia/commit/d39073e405921a64f2078aa9e6a31380bc9f5e55.
## PR Checklist:

- [x] Family categorization tags collected from the type design team with the Categories Form
~- [ ] Minisite_url definition in the METADATA.pb file for commissioned projects~
~- [ ] Primary_script definition in the METADATA.pb file for all projects that have a primary non-Latin based language support target~
- [ ] Fontbakery checks are reviewed and failing checks are resolved in collaboration with the upstream font development team
- [ ] Diffenator2 regression checks for revisions on all projects that are currently in production
- [ ] Designers bio info have to be present in the designer catalog (at least an issue should be opened for tracking this, if they are not)
- [ ] Check designers order in metadata.pb, since the first one of the list appears as “principal designer”
- [ ] Social media formatted visual assets for all new commissioned projects in the Drive directory, communicate with the repository Maintainer so that they can push this content to the Social Media tracker spreadsheet
- [ ] Social media content draft for all new commissioned projects in the Drive directory and Social Media tracker spreadsheet, communicate with the repository Maintainer so that they can push this content to the Social Media tracker spreadsheet

### CODE REVIEW

File: ofl/eduqldhandprecursive/METADATA.pb [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    source_file: "fonts/variable/EduQLDHandPrecursive[wght].ttf"
+    dest_file: "EduQLDHandPrecursive[wght].ttf"
+  }
+  files {
+    source_file: "DESCRIPTION.en_us.html"
+    dest_file: "DESCRIPTION.en_us.html"
+  }
+  branch: "main"
+}
+display_name: "Edu QLD Hand Precursivecursive"
```
Comment (vv-monsalve) [inline ofl/eduqldhandprecursive/METADATA.pb:42]: > Precursivecursive
I think there is a typo here, but I rather ask you to confirm about it

**Comment by vv-monsalve** [comment]
> 🔥 **FAIL** Ensure the font supports case swapping for all its glyphs.
🔥 FAIL The following glyphs lack their case-swapping counterparts:

Screen Shot 2024-04-04 at 20 25 17

@simoncozens, @EbenSorkin, it seems some of the reported cases are in the font `Ĕ` `Ĭ`, while others are actually missing `ŀ`.

File: ofl/eduqldhandprecursive/METADATA.pb (hunk 1 above)
Comment (EbenSorkin) [reply ofl/eduqldhandprecursive/METADATA.pb:42]: Yes, it should either be Precursive or cursive

Comment (vv-monsalve) [reply ofl/eduqldhandprecursive/METADATA.pb]: Thanks. Changed to "Precursive".

**Pull request closed without merging**